### PR TITLE
update incremental build interface.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @ansyral @superyyrrzz @hellosnow @qinezh 
#449

### CODE REVIEW

**Comment by qinezh** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Plugins/ISupportIncrementalDocumentProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        /// <summary>
+        /// Save intermediate model to stream.
+        /// </summary>
+        /// <param name="model">The model to save.</param>
+        /// <param name="stream">The stream for saving.</param>
+        void SaveIntermediateModel(FileModel model, Stream stream);
+        /// <summary>
+        /// Load intermediate model from stream.
+        /// </summary>
+        /// <param name="stream">The stream contain saved model.</param>
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Plugins/ISupportIncrementalDocumentProcessor.cs]: >contain [](start = 44, length = 7)

minor: containing

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Comment by ansyral** [comment]
:shipit:

**Comment by vwxyzh** [comment]
@herohua

**Comment by herohua** [comment]
:+1:

**Pull request merged**